packages/contracts/test/JwtRegistry/JwtRegistryBase.t.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+import {JwtRegistry} from "../../src/utils/JwtRegistry.sol";
+import {JwtRegistryTestBase} from "./JwtRegistryBase.t.sol";
+
+contract JwtRegistryTestBase is Test {
+    bytes32 publicKeyHash =
+        0x0ea9c777dc7110e5a9e89b13f0cfc540e3845ba120b2b6dc24024d61488d4788;
+    string kidIssAzpString = "12345|https://example.com|client-id-12345";
+    JwtRegistry jwtRegistry;
+
+    address deployer = vm.addr(1);
+
+    constructor() {}
+
+    function setUp() public virtual {
+        // Create jwt dkim registry
+        jwtRegistry = new JwtRegistry(deployer);
+        jwtRegistry.setDKIMPublicKeyHash(kidIssAzpString, publicKeyHash);
+    }
```

Comment:
@wshino I see that you are just setting the mapping but not checking if it was set correctly or not. Can you please also check if the mapping was set correctly? 

Also @SoraSuegami This flow will change once we have the Chainlink Functions integrated. Can you please explain @wshino how that would work?

Reply:
@shreyas-londhe Added.
ea99450c82fbc745afa5e25ab395142e58636964

Reply:
@shreyas-londhe Could you please check my question here?

Reply:
So before checking if the jwt pubkey is valid or not, we should call the chainlink function to fill any new keys until the last call. Not sure if this should be done manually or at every verification (based on costs). And yes, the contract setting the key should be jwtRegistry

Reply:
Thank you for your answer!
First of all, I don't think main contract implementations such as `JwtRegistry` in this repo should depend on any specific oracle (chainlink/icp/avs).
In the oracle-agnostic design, the owner of `JwtRegistry` will be a contract which updates the dkim public key hash, and that contract verifies that there are enough confirmations from the oracle committee parties.

> Not sure if this should be done manually 

The relayer will call the update function on the oracle only when the update is necessary.